### PR TITLE
Adnfx2 spanish translation

### PR DESCRIPTION
**Pull request by mbeylin**

### CODE REVIEW

**Comment by adnfx2** [comment]
Hi there! I highly suggest to complete the internationalization of some react components that are not 100% internationalized, If you don't mind I could help with that.

**Comment by molecula451** [comment]
Hello, there are also some words here on the on the file. That could be update and example. Word "Balance" still sound in english as "Balance" the most accurate in spanish translate for the Hispanic community could be "Saldo"

**Comment by mbeylin** [comment]
Hey @adnfx2 I've added a couple new lines to es_ES that need translating based on Corwin's review:
- (line 41) `Discard Changes`
- (line 42) `Upload New Photo`
- (line 866) `Stages` (all of them)

I'm also not sure about why the brackets exist as Corwin noted, so I'd be curious to hear

**Comment by adnfx2** [comment]
Hey everyone!, about the parentheses, I'm agreed that that component would look much more better without any parentheses at all, now, I used mainly because in spanish that translations became a bit odd to read, Here the reasoning:
-Look at "Intermediate Difficulty" in English this is completely fine, however, in spanish the only possible result for that translation is "Intermedio Dificultad" this is a bit odd to read in spanish since it looks like those words swapped their positions. It would be like reading "carpet red" instead of "red carpet"(weird right?), it's not a huge deal, I could just remove those parentheses and probably the user would infer what's going on. I always find this kind of things in other translated websites. BTW this same scenario occurs with "Private Bounty". 
-Now let's look at "a month restante(s)", Why I put parentheses there?, I did it mainly because I couldn't pluralize "restante(s)" in its current form, there wasn't a obvious way to do it since I'm not receiving any number to select the proper plural form, Ideally I should be capable of selecting "restante" for singular and "restantes" for plural based on the date number.
-Finally let's look at "entrega(s)", I was able to pluralize this, but because his upper friend "a month restante(s)" was using parentheses I thought It would better to give him some company by using parentheses here too. It's important to mention -again- that this translation can be successfully pluralize, I just did this to be a bit consistent. 

To summarize I would prefer not to use any  parentheses at all, I think that any hispanic user will be able to understand what's going on without any parentheses, It would just look a bit funny to read :). 

Hey molecula thanks for your suggestion, "saldo" and "balance" both are completely ok, I would say that this is highly subjective. I would like to hear any other suggestions you may have.

To Mark, I'll add the proper translations soon for those additions.

**Comment by mbeylin** [comment]
@corwinharrell I've made adjustments to the PR and added in @adnfx2's new translations for all of the copy. At this point everything is fixed (including the use of parentheses), the only remaining issue being the (difficulty) problem we have.

In my opinion we should keep the parentheses there, since it's the only place they're used, and I suspect it's going to get very very complicated for us to begin adjusting the styling of our components based on different languages ordering adjectives differently than in english. This solution seems sufficient and scalable, but I'm curious to hear your thoughts.

**Comment by adnfx2** [comment]
> @corwinharrell I've made adjustments to the PR and added in @adnfx2's new translations for all of the copy. At this point everything is fixed (including the use of parentheses), the only remaining issue being the (difficulty) problem we have.
> 
> In my opinion we should keep the parentheses there, since it's the only place they're used, and I suspect it's going to get very very complicated for us to begin adjusting the styling of our components based on different languages ordering adjectives differently than in english. This solution seems sufficient and scalable, but I'm curious to hear your thoughts.

Well, Now I thought a bit about this and I would removed, since it's not big deal, I'm pretty sure the user will understand the context, and the component won't look weird.

**Comment by corwinharrell** [comment]
@mbeylin I place my full trust in @adnfx2 😄

**Comment by molecula451** [comment]
Hello guys. I must say that the pluralization and taking in count the parentheses which @adnfx2 contribute it to it  are very healthy for a spanish perspective. Where in cases e.g  

[https://user-images.githubusercontent.com/6142777/61334115-1f930400-a7f7-11e9-85c1-73ac2c83b2e1.png](url)

The parentheses are being clearly, referring to the difficulty. In this case Principiante (in english Beginner) while on the english side it's viewed as a sentence Intermediate Difficulty or in spanish (Dificultad Intermedia) but because the syntax it'd be (Intermedia Dificultad) which in this case it points out to what @adnfx2 said about reading "carpet red" instead of "red carpet" which in fact it's obvisouly weird for english speakers.

**Comment by molecula451** [comment]
Hello guys. I must say that the pluralization and taking in count the parentheses which @adnfx2 contribute it to it are very healthy for a spanish perspective. Where in cases e.g

61334115-1f930400-a7f7-11e9-85c1-73ac2c83b2e1


The parentheses are being clearly, referring to the difficulty. In this case Principiante (in english Beginner) while on the english side it's viewed as a sentence Intermediate Difficulty or in spanish (Dificultad Intermedia) but because the syntax it'd be (Intermedia Dificultad) which in this case it points out to what @adnfx2 said about reading "carpet red" instead of "red carpet" which in fact it's obvisouly weird for english speakers.
